node/pkg/cni/token_watch.go

```diff
@@ -238,7 +238,7 @@ func Run() {
 }
 
 // CNIServiceAccountName returns the name of the serviceaccount to use for the CNI plugin token request.
-// This can be set via the CALICO_CNI_SERVICE_ACCOUNT environment variable, and defaults to "calico-node" otherwise.
+// This can be set via the CALICO_CNI_SERVICE_ACCOUNT environment variable, and defaults to "calico-cni-plugin" otherwise.
```

Comment:
I think it would be good to add a unit test or a new assertion to `token_watch_test.go` to verify that the returned token has the correct serviceaccount information in it. 

You can take a ServiceAccount and base64 decode it. It's a JWT that will include information about the service account to which the token is bound.